third_party/packages/split/lib/split.dart

```diff
@@ -40,11 +42,22 @@ class _SplitOptions {
   external List<num> get sizes;
 
   external List<num> get minSize;
+
+  /// minSize is ignored unless this property is set.
+  external bool get expandToMin;
```

Comment:
nit: optional - maybe rename to `splitAtMin`. Expanding to a minimum just seems a little funky. Or maybe `collapseToMin`

Reply:
these are names from the JS library so I can't change them.